### DISCUSSION TITLE
CIS version 1.2.0 and 1.3.0 support on OpenShift

### DISCUSSION THREAD

**Discussion by deebhatia** [aquasecurity/kube-bench#1457] [Development]
Hello,

Any plans to introduce version [1.2.0](https://workbench.cisecurity.org/benchmarks/10252) and [1.3.0](https://workbench.cisecurity.org/benchmarks/7280) for RedHat OpenShift Container Platform?
The current available version 1.1.0 has some incompatibility with newer releases of RHOCP and give false failures.

BR,
Deepanshu

**Comment by deebhatia** [comment]
Any updates? Even 1.4 and 1.5 is also available at CIS end.

**Comment by chen-keinan** [comment]
@deebhatia contributions are welcome

**Reply by deebhatia** [reply]
@chen-keinan Opened a PR for the latest OpenShift CIS version.
https://github.com/aquasecurity/kube-bench/pull/1682
